packages/launcher/lib/known-browsers.ts

```diff
@@ -1,5 +1,24 @@
 import type { Browser, BrowserValidatorResult, FoundBrowser } from '@packages/types'
 
+const firefoxValidatorFn = (browser: FoundBrowser, platform: NodeJS.Platform): BrowserValidatorResult => {
+  try {
+    if (browser.majorVersion) {
+      const [majorVersion] = browser.majorVersion.split('.').map(Number)
```

Comment:
Is the `browser.majorVersion` not already parsed?

Reply:
I think I originally had `version` here and then updated to major version but didn't update the code. oops! Will get this updated!

Reply:
updated in 425bbaa4e1